ResearchKit/Consent/ORKConsentLearnMoreViewController.m

```diff
@@ -69,10 +81,17 @@ - (void)viewDidLoad {
     _webView.scrollView.scrollIndicatorInsets = (UIEdgeInsets){.left = -horizMargin, .right = -horizMargin};
     _webView.opaque = NO; // If opaque is set to YES, _webView shows a black right margin during transition when modally presented. This is an artifact due to disabling clipsToBounds to be able to show the scroll indicator outside the view.
     
-    [_webView loadHTMLString:self.content baseURL:ORKCreateRandomBaseURL()];
+    if(_contentRequest)
+    {
```

Comment:
Code style (here and everywhere):

We want
```
if (a) {
..
} else {
...
}
```

Note spaces before and after `else`, and bracket positioning